tests/tck/features/expression/Predicate.feature

```diff
@@ -49,6 +49,118 @@ Feature: Predicate
+      | ("Marco Belinelli" :player{age: 32, name: "Marco Belinelli"})                                               |
+      | ("LeBron James" :player{age: 34, name: "LeBron James"})                                                     |
+      | ("Rajon Rondo" :player{age: 33, name: "Rajon Rondo"})                                                       |
+      | ("Tiago Splitter" :player{age: 34, name: "Tiago Splitter"})                                                 |
+      | ("Shaquile O'Neal" :player{age: 47, name: "Shaquile O'Neal"})                                               |
+      | ("Boris Diaw" :player{age: 36, name: "Boris Diaw"})                                                         |
+      | ("Aron Baynes" :player{age: 32, name: "Aron Baynes"})                                                       |
+      | ("Paul Gasol" :player{age: 38, name: "Paul Gasol"})                                                         |
+      | ("Kristaps Porzingis" :player{age: 23, name: "Kristaps Porzingis"})                                         |
+      | ("Klay Thompson" :player{age: 29, name: "Klay Thompson"})                                                   |
+      | ("Stephen Curry" :player{age: 31, name: "Stephen Curry"})                                                   |
+      | ("Kyle Anderson" :player{age: 25, name: "Kyle Anderson"})                                                   |
+      | ("Kevin Durant" :player{age: 30, name: "Kevin Durant"})                                                     |
+      | ("Amar'e Stoudemire" :player{age: 36, name: "Amar'e Stoudemire"})                                           |
+      | ("Marc Gasol" :player{age: 34, name: "Marc Gasol"})                                                         |
+      | ("Ben Simmons" :player{age: 22, name: "Ben Simmons"})                                                       |
+      | ("JaVale McGee" :player{age: 31, name: "JaVale McGee"})                                                     |
+      | ("Joel Embiid" :player{age: 25, name: "Joel Embiid"})                                                       |
+      | ("Blake Griffin" :player{age: 30, name: "Blake Griffin"})                                                   |
+      | ("Damian Lillard" :player{age: 28, name: "Damian Lillard"})                                                 |
+    When executing query:
+      """
+      MATCH(n:player) WHERE EXISTS("abc")
+      RETURN n.name AS name ORDER BY name LIMIT 10
+      """
+    Then a SyntaxError should be raised at runtime: The exists only accept LabelAttribe, Attribute and Subscript
+    Then drop the used space
+
+  Scenario: exists with dynamic map in variable length MATCH
+    Given a graph with space named "nba"
+    When executing query:
+      """
+      MATCH(v:player{name:"Tim Duncan"})-[e:like|serve*2]->(v2)
+      RETURN DISTINCT e, ALL(e IN e WHERE EXISTS(e['likeness']))
```

Comment:
Is is valid for this same variable name `e` as list variable name `e` @jievince ?

Reply:
yes, it's valid. Because after the parser, the inner var has been rewrited to another name